### DISCUSSION TITLE
While adding a Tag, show a dropdown of existing tags

### DISCUSSION THREAD

**Discussion by neadfiles** [Kovah/LinkAce#701] [Ideas & Feature Requests]
Hopefully this is an easy request!

When I am adding a tag I don't always remember what my existing tags are. Currently, I have to choose All Tags from the menu and that takes me away from the insert tag field.

It would even be more helpful if an existing tag list popped up and we could add tags by selection.

Thanks for your continued work (in your free time!) on Linkace.

**Comment by hugofant** [comment]
Hi,
have installed Linkace today and this was my first noticed "missing feature" ;-)
Is there a chance, that it will be implemented in near future?

**Reply by Kovah** [reply]
It's actually a performance consideration. It could easily be done for people with like 30 tags, but what about having a 1000+ tags?
I would rather display new newest 10-20 tags or lists. This was already requested somewhere else.

**Reply by James000r** [reply]
Hi Kovah. I am refering to the browser-plugin in:
1. What about providing the last 10 lists or tags used?
2. Would a function to jump / open to all list in LinkAce thinkable?
Thanks and cheers,
James

**Reply by Kovah** [reply]
Linkace 2 has all tags and lists enabled in the dropdown. There's actually no considerable performance impact for 1000 of either the two, so I just went with that option.
